test/utils/orderWrapper.ts

```diff
@@ -4,46 +4,46 @@ import * as ethUtil from "ethereumjs-util";
 import { BigNumber } from "bignumber.js";
 import { Address, Bytes32, UInt } from "../../types/common.js";
 import {
-  ORDER_TYPE,
+  EXCHANGES,
 } from "../utils/constants";
 
-export function generateExchangeOrdersHeader(
-  orderCount: UInt
+export function generateOrdersDataForOrderCount(
+  orderCount: number
 ): Bytes32 {
-  return ethUtil.bufferToHex(
-    ethUtil.setLengthLeft(
-      ethUtil.toBuffer(orderCount), 32)
-    );
+  const exchangeOrderDatum: Buffer[] = [];
+  _.times(orderCount, (index) => {
+    const exchange = _.sample(EXCHANGES);
+    exchangeOrderDatum.push(paddedBufferForData(exchange));
+
+    const orderLength = _.random(120, 160)
```

Comment:
Hoping to sub any kind of util function we write later that generates a real exchange order that mirrors what `setProtocol.js` will return